### PR TITLE
[Backport 2.x] Patch bump for scala dependency

### PR DESCRIPTION
**Pull request by scrawfor99**
Signed-off-by: Stephen Crawford <steecraw@amazon.com>

Backports Commit #2163 into 2.x.

### Check List
- [ ] ~New functionality includes testing~
- [ ] ~New functionality has been documented~
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by peternied** [comment]
@scrawfor99 Looks like there was a merge conflict that needs to be resolved manually

File: build.gradle [hunk 1]
```diff
@@ -221,11 +221,12 @@ bundlePlugin {
     }
 }
 
+<<<<<<< HEAD
```
Comment (cliu123) [inline build.gradle:224]: Please remove this line.

**Review by cwperks — APPROVED** [review]

**Comment by cwperks** [comment]
@scrawfor99 Please update the title to `[Backport 2.x] Patch bump for scala dependency`

**Comment by scrawfor99** [comment]
The linked issue about the Woodstox vulnerability mentions that at the time of this issue there was no current updated solution: https://github.com/opensearch-project/security/issues/2104

I checked and this continues to be the case. I believe for now we are fine to backport the changes.

**Comment by scrawfor99** [comment]
> @scrawfor99 Please update the title to `[Backport 2.x] Patch bump for scala dependency`

Done sorry.

**Review by peternied — APPROVED** [review]

**Pull request merged**